js/client.js

```diff
@@ -2594,10 +2594,14 @@ function toId() {
 
 			buf += '<p class="buttonbar">';
 			if (userid === app.user.get('userid') || !app.user.get('named')) {
-				buf += '<button disabled>Challenge</button> <button disabled>Chat</button>';
+				buf += '<button disabled>Challenge</button>';
 				if (userid === app.user.get('userid')) {
+					buf += ' <button name="pm">Chat</button>';
```

Comment:
```suggestion
					buf += ' <button name="pm">Chat self</button>';
```

Just to make it clearer.